src/configuration/app-containers.md

```diff
@@ -316,20 +316,45 @@ web:
             passthru: "/index.php"
             index:
                 - index.php
+            # No caching, unless the PHP script sends a cache header.
```

Comment:
I thought `expires` only applies to static files, and doesn't affect scripts at all?